src/code/bank20.asm

```diff
@@ -4961,11 +4961,11 @@ IntroCopyAndBlendColor::
     dec  bc                                       ; $6B92: $0B
     dec  hl                                       ; $6B93: $2B
 
-    ; Switch back to RAM bank 0
+    ; Switch back to RAM bank 1
```

Comment:
No, this switches the WRAM (upper bit, which is bankable on GBC) to WRAM bank 0.

Reply:
@daid I changed this back based on your comment, but on second thought I'm wondering if you're mistaken here or whether I'm misunderstanding something? According to [Pan Docs](https://gbdev.io/pandocs/CGB_Registers.html#ff70--svbk-cgb-mode-only-wram-bank) and `hardware.inc`, writing 0 to `rSVBK` maps WRAM bank 1 to `$D000`-`$DFFF`; WRAM bank 0 is always mapped to `$C000`-`$CFFF`.

Reply:
Oh, damn, you are right: https://gbdev.io/pandocs/CGB_Registers.html#ff70--svbk-cgb-mode-only-wram-bank
I though there where 8 banks that could be mapped to the upper area, but there are only 7.